### DISCUSSION TITLE
I can't receive the verification email for the new email address I added...for already two days.

### DISCUSSION THREAD

**Discussion by StellaLiu-gogogo** [community/community#49677] [GitHub Education]
### Select Topic Area

Question

### Body

I need to add my new university email address to activate the github copilot for teacher. 
However, I've tried two days and clicking on the resend button several times but no email arrives even in the junk mail or quarantine center of my university email box
Any suggestions how to verify a new email address except just waiting to test my luck?
I see in the forum I'm not the only one. Anyone has already solved the problem?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @StellaLiu-gogogo ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
